operators/monitoring/values.tpl.yaml

```diff
@@ -0,0 +1,35 @@
+grafana:
+  envValueFrom:
+    GF_AUTH_GENERIC_OAUTH_CLIENT_ID:
+      secretKeyRef:
+        name: grafana-sso
+        key: client-id
+    GF_AUTH_GENERIC_OAUTH_CLIENT_SECRET:
+      secretKeyRef:
+        name: grafana-sso
+        key: client-secret
+  grafana.ini:
+    auth.generic_oauth:
+      name: Dex
+      enabled: true
+      client_id: $__env{GF_AUTH_GENERIC_OAUTH_CLIENT_ID}
+      client_secret: $__env{GF_AUTH_GENERIC_OAUTH_CLIENT_SECRET}
+      scopes: openid email profile groups offline_access
+      auth_url: https://dex.${DNS_ZONE}/auth
+      token_url: https://dex.${DNS_ZONE}/token
+      api_url: https://dex.${DNS_ZONE}/userinfo
+    server:
+      root_url: https://%(domain)s
```

Comment:
How does this get set?

Reply:
`https://%(domain)s` the domain comes from `server: domain` in the helm chart: https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml#L799-L800